### PR TITLE
[DOCS] Revises and reorganizes content for installing additional dependencies per the GX 1.0 ToC

### PR DESCRIPTION
**Pull request by Rachel-Reverie**
## Description
- Adds "Install additional dependencies" topic per the revised 1.0 ToC
- Revises content provided for clarity.

## Definition of done
- [X] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [ ] Appropriate tests and docs have been updated

### CODE REVIEW

File: docs/docusaurus/docs/core/_core_components/prerequisites/_python_installation.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 import VersionedLink from '@site/src/components/VersionedLink';
 import GxData from '../../../components/_data.jsx';
 
-<span><VersionedLink to='/core/installation_and_setup/set_up_a_python_environment'>An installation of Python, version {GxData.min_python} to {GxData.max_python}</VersionedLink></span>
\ No newline at end of file
+<span><VersionedLink to='/core/set_up_a_gx_environment/install_python'>An installation of Python, version {GxData.min_python} to {GxData.max_python}</VersionedLink></span>
```
Comment (kwcanuck) [inline docs/docusaurus/docs/core/_core_components/prerequisites/_python_installation.md]:
```suggestion
<span><VersionedLink to='/core/set_up_a_gx_environment/install_python'>Python version {GxData.min_python} to {GxData.max_python}</VersionedLink></span>
```

File: docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_amazon_s3.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+## Prerequisites
+
+- The AWS CLI. See [Installing or updating the latest version of the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html).
+- AWS credentials. See [Configuring the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html).
+- <PrereqPythonInstalled/>
+- <RecommendedVirtualEnvironment/>
+
+## Installation
+
+Python interacts with AWS through the `boto3` library. {GxData.product_name} makes use of this library in the background when working with AWS. Although you won't use `boto3` directly, you'll need to install it in your python environment.
```
Comment (kwcanuck) [inline docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_amazon_s3.md]:
```suggestion
Python interacts with AWS through the `boto3` library. {GxData.product_name} and uses the library in the background when working with AWS. Although you won't use `boto3` directly, you must install it in your Python environment.
```

File: docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_amazon_s3.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+   :::tip
+
+   If the `python -m pip install boto3` does not work, try:
+
+   ```bash title="Terminal input"
+   python3 -m pip install boto3
+   ```
+   
+   If neither of these pip commands work, you may need to verify that [Python is installed correctly](core/set_up_a_gx_environment/install_gx.md).
```
Comment (kwcanuck) [inline docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_amazon_s3.md]:
```suggestion
   If these `pip` commands work, verify that [Python is installed correctly](core/set_up_a_gx_environment/install_gx.md).
```

File: docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_amazon_s3.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+   :::
+
+3. Run the following terminal command to verify your AWS credentials are properly configured:
+
+   ```bash title="Terminal input"
+   aws sts get-caller-identity
+   ```
+
+   When your credentials are properly configured, your `UserId`, `Account`, and `Arn` are returned. If your credentials are not configured correctly, an error message appears. If you received an error message, or you couldn't verify your credentials, see Amazon's documentation for [Configure the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html).
```
Comment (kwcanuck) [inline docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_amazon_s3.md:53]:
```suggestion
   When your credentials are properly configured, your `UserId`, `Account`, and `Arn` are returned. If your credentials are not configured correctly, an error message appears. If you received an error message or you couldn't verify your credentials, see [Configure the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html).
```

File: docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_amazon_s3.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+  
+4. Install the Python dependencies for AWS S3 support.
+
+   :::info
+   <InfoUsingAVirtualEnvironment/>
+   :::
+
+   Run the following terminal command to install the optional dependencies required by GX to work with AWS S3:
```
Comment (kwcanuck) [inline docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_amazon_s3.md]:
```suggestion
   Run the following terminal command to install the optional dependencies required by {GxData.product_name} to work with AWS S3:
```
Correct all instances.

File: docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_azure_blob_storage.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
+import GxData from '../../_core_components/_data.jsx';
+import PrereqPythonInstalled from '../../_core_components/prerequisites/_python_installation.md';
+import RecommendedVirtualEnvironment from '../../_core_components/prerequisites/_recommended_virtual_environment.md';
+import InfoUsingAVirtualEnvironment from '../../_core_components/admonitions/_if_you_are_using_a_virtual_environment.md';
+
+Azure Blob Storage stores unstructured data on the Microsoft cloud data storage platform. To validate Azure Blob Storage data with GX Core you install additional Python libraries and define a connection string.
```
Comment (kwcanuck) [inline docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_azure_blob_storage.md]:
```suggestion
Azure Blob Storage stores unstructured data on the Microsoft cloud data storage platform. To validate Azure Blob Storage data with {GxData.product_name} you install additional Python libraries and define a connection string.
```

File: docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_azure_blob_storage.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+## Installation
+  
+1. Install the Python dependencies for AWS S3 support.
+
+   :::info
+   <InfoUsingAVirtualEnvironment/>
+   :::
+
+   Run the following code to install {GxData.product_name} with the additional Python libraries needed to work with Azure Blob Storage:
```
Comment (kwcanuck) [inline docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_azure_blob_storage.md]:
```suggestion
   2. Run the following code to install {GxData.product_name} with the additional Python libraries needed to work with Azure Blob Storage:
```
Update the following numbered procedure steps.

File: docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_azure_blob_storage.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+
+   ```bash title="Terminal input"
+   python -m pip install 'great_expectations[azure]'
+   ```
+
+2. Configure your Azure Blob Storage credentials.
+
+   You can manage your credentials by storing them as environment variables.  To do this, enter `export ENV_VARIABLE_NAME=env_var_value` in the terminal or add the equivalent command to your `~/.bashrc` file. For example:
```
Comment (kwcanuck) [inline docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_azure_blob_storage.md]:
```suggestion
   You can manage your credentials by storing them as environment variables.  To do this, enter `export ENV_VARIABLE_NAME=env_var_value` in the terminal or add the equivalent command to your `~/.bashrc` file. For example:
```
Move this so it follows _Configure your Azure Blob Storage credentials_.

File: docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_azure_blob_storage.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+
+   ```title="Terminal input"
+   export AZURE_STORAGE_CONNECTION_STRING="DefaultEndpointsProtocol=https;EndpointSuffix=core.windows.net;AccountName=<YOUR-STORAGE-ACCOUNT-NAME>;AccountKey=<YOUR-STORAGE-ACCOUNT-KEY>"
+   ```
+
+   When entering this command, replace `<YOUR-STORAGE-ACCOUNT-NAME>` and `<YOUR-STORAGE-ACCOUNT-KEY>` with your Azure Blob Storage account values.
```
Comment (kwcanuck) [inline docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_azure_blob_storage.md]:
```suggestion
   Replace `<YOUR-STORAGE-ACCOUNT-NAME>` and `<YOUR-STORAGE-ACCOUNT-KEY>` with your Azure Blob Storage account values.
```
Always look for opportunities to trim text.

File: docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_google_cloud_platform.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
+import GxData from '../../_core_components/_data.jsx';
+import PrereqPythonInstalled from '../../_core_components/prerequisites/_python_installation.md';
+import RecommendedVirtualEnvironment from '../../_core_components/prerequisites/_recommended_virtual_environment.md';
+import InfoUsingAVirtualEnvironment from '../../_core_components/admonitions/_if_you_are_using_a_virtual_environment.md';
+
+To validate Google Cloud Platform (GCP) data with {GxData.product_name}, you create your GX Python environment, install
+{GxData.product_name} locally, and then configure the necessary dependencies.
```
Comment (kwcanuck) [inline docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_google_cloud_platform.md]:
```suggestion
{GxData.product_name} locally, and then configure the dependencies.
```
Are some dependencies unnecessary? Also, remove the hard return.

File: docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_google_cloud_platform.md [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+
+## Prerequisites
+
+- A [GCP service account](https://cloud.google.com/iam/docs/service-account-overview) with permissions to access GCP resources and storage Objects.
+- The `GOOGLE_APPLICATION_CREDENTIALS` environment variable is set. See the Google documentation [Set up Application Default Credentials](https://cloud.google.com/docs/authentication/provide-credentials-adc).
```
Comment (kwcanuck) [inline docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_google_cloud_platform.md:11]:
```suggestion
- The `GOOGLE_APPLICATION_CREDENTIALS` environment variable is set. See [Set up Application Default Credentials](https://cloud.google.com/docs/authentication/provide-credentials-adc). 
```
You don't need to define the owner of the linked documentation. It is obvious when a user selects the link.

File: docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_google_cloud_platform.md [hunk 12]
```diff
@@ -0,0 +1,37 @@
...
+import InfoUsingAVirtualEnvironment from '../../_core_components/admonitions/_if_you_are_using_a_virtual_environment.md';
+
+To validate Google Cloud Platform (GCP) data with {GxData.product_name}, you create your GX Python environment, install
+{GxData.product_name} locally, and then configure the necessary dependencies.
+
+## Prerequisites
+
+- A [GCP service account](https://cloud.google.com/iam/docs/service-account-overview) with permissions to access GCP resources and storage Objects.
+- The `GOOGLE_APPLICATION_CREDENTIALS` environment variable is set. See the Google documentation [Set up Application Default Credentials](https://cloud.google.com/docs/authentication/provide-credentials-adc). 
+- Google Cloud API authentication is set up. See the Google documentation [Set up authentication](https://cloud.google.com/storage/docs/reference/libraries#authentication).
```
Comment (kwcanuck) [inline docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_google_cloud_platform.md:12]:
```suggestion
- Google Cloud API authentication is set up. See [Set up authentication](https://cloud.google.com/storage/docs/reference/libraries#authentication).
```

File: docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_google_cloud_platform.md [hunk 13]
```diff
@@ -0,0 +1,37 @@
...
+- <PrereqPythonInstalled/>
+- <RecommendedVirtualEnvironment/>
+
+## Installation
+
+1. Ensure your GCP credentials are correctly configured.
```
Comment (kwcanuck) [inline docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_google_cloud_platform.md]: This needs a rewrite.

1. Ensure your GCP credentials are correctly configured. This process includes the following:

   - Creating a Google Cloud Platform (GCP) service account.
   - Setting the ``GOOGLE_APPLICATION_CREDENTIALS`` environment variable,
   - Verifying authentication by running a [Google Cloud Storage client](https://cloud.google.com/storage/docs/reference/libraries) library script.
  
    For more information, see [Authenticate for using client libraries](https://cloud.google.com/docs/authentication/getting-started).

File: docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_google_cloud_platform.md [hunk 14]
```diff
@@ -0,0 +1,37 @@
...
+
+   The Google Cloud Platform documentation describes how to verify [authentication for the Google Cloud API](https://cloud.google.com/docs/authentication/getting-started), which includes:
+
+   - Creating a Google Cloud Platform (GCP) service account.
+   - Setting the ``GOOGLE_APPLICATION_CREDENTIALS`` environment variable,
+   - Verifying authentication by running a [Google Cloud Storage client](https://cloud.google.com/storage/docs/reference/libraries) library script.
+
+2. Install the Python dependencies for GCP support.
```
Comment (kwcanuck) [inline docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_google_cloud_platform.md:26]:
```suggestion
2. 
```
Move _Run the following terminal command_ so it follows the numbered step.

File: docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_sql.md [hunk 15]
```diff
@@ -0,0 +1,58 @@
+import GxData from '../../_core_components/_data.jsx';
+import PrereqPythonInstalled from '../../_core_components/prerequisites/_python_installation.md';
+import RecommendedVirtualEnvironment from '../../_core_components/prerequisites/_recommended_virtual_environment.md';
+import InfoUsingAVirtualEnvironment from '../../_core_components/admonitions/_if_you_are_using_a_virtual_environment.md';
+
+To validate data stored on SQL databases with {GxData.product_name}, you create your GX Python environment, install {GxData.product_name} locally, and then configure the necessary dependencies.
```
Comment (kwcanuck) [inline docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_sql.md:7]:
```suggestion
To validate data stored on SQL databases with {GxData.product_name}, you create your GX Python environment, install {GxData.product_name} locally, and then configure the dependencies.
```

File: docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_sql.md [hunk 16]
```diff
@@ -0,0 +1,58 @@
...
+
+## Prerequisites
+
+- <PrereqPythonInstalled/>
+- <RecommendedVirtualEnvironment/>
+
+## Installation
+
+1. Run the following pip command to install optional dependencies for `SQLAlchemy`:
```
Comment (kwcanuck) [inline docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_sql.md]:
```suggestion
1. Run the following `pip` command to install optional dependencies for `SQLAlchemy`:
```

File: docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_sql.md [hunk 17]
```diff
@@ -0,0 +1,58 @@
...
+
+   ```bash title="Terminal input"
+   python -m pip install 'great_expectations[sqlalchemy]'
+   ```
+
+   To install dependencies for a different SQL dialect, use the corresponding command from [SQL dialect dependency commands](#sql-dialect-dependency-commands).
```
Comment (kwcanuck) [inline docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_sql.md]:
```suggestion
   To install dependencies for a different SQL dialect, use the [SQL dialect dependency commands](#sql-dialect-dependency-commands).
```

File: docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_sql.md [hunk 18]
```diff
@@ -0,0 +1,58 @@
...
+
+2. Configure an environment variable with the credentials to access your SQL database.
```
Comment (kwcanuck) [inline docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_sql.md]: See previous edits.

File: docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_sql.md [hunk 19]
```diff
@@ -0,0 +1,58 @@
...
+
+   You can manage your credentials by storing them as environment variables.  To do this, enter `export ENV_VARIABLE_NAME=env_var_value` in the terminal or add the equivalent command to your `~/.bashrc` file. For example:
+
+   ```bash title='Terminal input'
+   export MY_DB_PW=<MY_PASSWORD>
+   ```
+
+   When entering this command, replace `<MY_PASSWORD>` with your SQL database password.
+
+   Once your password is stored in an environment variable, you can reference it in GX using string substitution.  For instance, if you were accessing a PostgreSql database, you could provide a connection string that uses `${MY_DB_PW}` to insert your password from the environment variable:
```
Comment (kwcanuck) [inline docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_sql.md]: This should be an optional step. For example:

3. Optional. Create a string substitution to reference the password environment variable. 

Always look for opportunities to reduce clutter and improve the procedural flow.

File: docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_sql.md [hunk 20]
```diff
@@ -0,0 +1,58 @@
...
+
+   ```bash title="Example PostgreSql connection string"
+    "postgresql+psycopg2://<username>:${MY_DB_PW}@<host>:<port>/<database>"
+   ```
+
+   :::info
```
Comment (kwcanuck) [inline docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_sql.md]: This is information that should be provided before a user creates the environment variables.

**Review by kwcanuck — APPROVED** [review]
A quick review. Feel free to implement or ignore the edits.

File: docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_google_cloud_platform.md (hunk 11 above)
Comment (Rachel-Reverie) [reply docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_google_cloud_platform.md:11]: It is, but that's after they've left our site.  It's generally considered good form to inform people in advance when clicking on a link will take them off of your site.

File: docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_sql.md (hunk 19 above)
Comment (Rachel-Reverie) [reply docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_sql.md]: I shifted this into an info admonition -- it's not _really_ a step in the current process, but it explains why they're performing the current step and what they'll be doing with it later.

**Pull request merged**